spec/features/random_peer_review_spec.rb

```diff
@@ -0,0 +1,124 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'random peer reviews', type: :feature, js: true do
+  let(:course) { create(:course) }
+  let(:campaign) { create(:campaign) }
+  let(:instructor) { create(:instructor) }
+  let(:student1) { create(:user) }
+  let(:student2) { create(:test_user) }
+
+  before do
+    create(:campaigns_course, campaign_id: campaign.id, course_id: course.id)
```

Comment:
It's slightly more concise to do `campaign: campaign, course: course` instead of explicitly using the IDs.